package.json

```diff
@@ -8,12 +8,16 @@
     "test": "nyc --reporter=lcov mocha",
     "release": "standard-version -a",
     "openshift": "nodeshift --dockerImage=registry.access.redhat.com/ubi8/nodejs-16",
-    "start": "node ."
+    "openshift:enable:trace": "nodeshift --deploy.env NODE_OPTIONS='--require ./tracing.js' --dockerImage=registry.access.redhat.com/ubi8/nodejs-16",
+    "start": "node .",
```

Comment:
I think that should be more specific to which files node should start.

Reply:
thanks for the review :+1:
This is defined on the `main` property: 
https://github.com/helio-frota/main-node-dot